WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -151,7 +151,7 @@ final class MainTabBarController: UITabBarController {
 
     private var posTabVisibilityChecker: POSTabVisibilityCheckerProtocol?
     private var posEligibilityCheckTask: Task<Void, Never>?
-    private var posCatalogSyncCoordinator: POSCatalogSyncCoordinatorProtocol?
+    private lazy var posSyncDispatcher = ForegroundPOSCatalogSyncDispatcher()
```

Comment:
I'm guessing this needs to be lazy because the POS tab visibility and eligibility? Otherwise we would be paying for initializing the sync dispatcher despite the merchant not being able to use POS.